client/src/app/appCtrl.js

```diff
@@ -15,8 +15,15 @@ angular.module('app').controller('appCtrl', ['$scope', '$http', '$timeout', 'app
       threshold : appConfig.MAX_FILE_SIZE,
       show : false,
       paused : false,
-      aborted : false
-    }
+      aborted : false,
+    },
+    highlighting : { // section for highlighting (anomalie) over a threshold
```

Comment:
I remember your comment about the `$scope`, but yes, I'd like it accessible through the UI. (we could expose only those used in UI, but imho it is better to keep the logically related variables in a same structure).